src/content-helper/excerpt-generator/excerpt-generator.scss

```
@@ -1,47 +1,55 @@
 @import "../common/css/variables";
 @import "../common/css/functions";
 
-.editor-post-excerpt__spinner {
+.editor-post-excerpt__loading_animation {
 	position: absolute;
-	top: to_rem(20px);
-	left: 0;
-	right: 0;
-	bottom: 0;
-	display: flex;
-	justify-content: center;
-	align-items: center;
-	background: rgba(255, 255, 255, 0.7); // semi-transparent white background.
-	z-index: 10;
+	top: to_rem(29px);
+	left: to_rem(9px);
 }
 
-.editor-post-excerpt__spinner.has-word-count {
-	bottom: to_rem(20px);
+.editor-post-excerpt__textarea {
+	margin-bottom: to_rem(8px);
+
+	textarea:read-only {
+		background-color: var(--Gutenberg-White, #fff);
+	}
 }
 
 .wp-parsely-excerpt-generator {
-	margin-top: to_rem(15px);
+	margin-top: to_rem(20px);
 
 	.wp-parsely-excerpt-generator-header {
 		align-items: center;
 		display: flex;
 		justify-content: flex-start;
-		margin-bottom: to_rem(10px);
 
 		.wp-parsely-excerpt-generator-header-label {
 			font-size: to_rem(11px);
-			font-weight: 500;
+			font-weight: 600;
 			line-height: 1.4;
 			text-transform: uppercase;
 			display: inline-block;
 			padding: 0;
 			margin-left: to_rem(5px);
+
+			span.beta-label {
+				padding-left: to_rem(6px);
+				color: var(--Gutenberg-Gray-700, #757575);
+			}
 		}
 	}
 
 	.wp-parsely-excerpt-generator-controls {
 		display: flex;
-		justify-content: center;
-		align-items: center;
-		gap: to_rem(15px);
+		gap: to_rem(8px);
+
+		button {
+			flex-grow: 1;
+			height: to_rem(40px);
+			justify-content: center;
+			margin-bottom: to_rem(8px);
+			margin-top: to_rem(8px);
+			padding: var(--grid-unit-10, 8px) var(--grid-unit-15, 12px);
```

In the other PRs I have been converting the fallback values to rem with `to_rem` as well. Not sure if it's strictly necessary, but that way we can keep consistency with using rem values even when the variable is undefined.

What do you think? 
```suggestion
			padding: var(--grid-unit-10, to_rem(8px)) var(--grid-unit-15, to_rem(12px));
```